tutorials/international/index.md

```diff
@@ -330,36 +454,38 @@ which defines all of the localizable strings used by the app.
 
 Rebuilding `l10n/messages_all.dart` requires two steps.
 
-1. With the app's root directory as the current directory, generate
+<ol markdown="1">
+<li markdown="1">With the app's root directory as the current directory, generate
 `l10n/intl_messages.arb` from `lib/main.dart`:
 
-{% commandline %}
-flutter pub pub run intl_translation:extract_to_arb --output-dir=lib/i10n lib/main.dart
-{% endcommandline %}
+{% prettify sh %}
+$ flutter pub pub run intl_translation:extract_to_arb --output-dir=lib/i10n lib/main.dart
+{% endprettify %}
 
 The `intl_messages.arb` file is a JSON format map with one entry for
 each `Intl.message()` function defined in `main.dart`. This
 file serves as a template for the the English and Spanish translations,
 `intl_en.arb` and `intl_es.arb`. These translations are created by you,
 the developer.
+</li>
 
-2. With the app's root directory as the current directory, generate
+<li markdown="1">With the app's root directory as the current directory, generate
 `intl_messages_<locale>.dart` for each `intl_<locale>.arb` file and
 `intl_messages_all.dart`, which imports all of the messages files:
 
-{% commandline %}
-flutter pub pub run intl_translation:generate_from_arb --output-dir=lib/l10n \
+{% prettify sh %}
+$ flutter pub pub run intl_translation:generate_from_arb --output-dir=lib/l10n \
    --no-use-deferred-loading lib/main.dart lib/l10n/intl_*.arb
-{% endcommandline %}
+{% endprettify %}
 
 The `DemoLocalizations` class uses the generated `initializeMessages()`
 function (defined in `intl_messages_all.dart`) to load the localized messages
 and `Intl.message()` to look them up.
-</aside>
+</ol>
 
-An instance of `DemoLocalizations` is created by a simple `LocalizationsDelegate`:
+An instance of DemoLocalizations is created by a simple LocalizationsDelegate:
```

Comment:
The text that follows (to the end) doesn't belong in the Appendix.  It looks like it all came from before the "Specifying the App’s supportedLocales Parameter" heading.